src/application.rs

```diff
@@ -208,25 +219,47 @@ pub trait ApplicationHandler {
 
     /// Emitted when the application has been suspended.
     ///
-    /// # Portability
+    /// See [`resumed()`][Self::resumed].
     ///
-    /// Not all platforms support the notion of suspending applications, and there may be no
-    /// technical way to guarantee being able to emit a `Suspended` event if the OS has
-    /// no formal application lifecycle (currently only Android, iOS, and Web do). For this reason,
-    /// Winit does not currently try to emit pseudo `Suspended` events before the application
-    /// quits on platforms without an application lifecycle.
+    /// ## Platform-specific
     ///
-    /// Considering that the implementation of `Suspended` and [`Resumed`] events may be internally
-    /// driven by multiple platform-specific events, and that there may be subtle differences across
-    /// platforms with how these internal events are delivered, it's recommended that applications
-    /// be able to gracefully handle redundant (i.e. back-to-back) `Suspended` or [`Resumed`]
-    /// events.
+    /// ### iOS
+    ///
+    /// On iOS, the [`suspended()`] method is called in response to an
+    /// [`applicationWillResignActive`] callback which means that the application is about to
+    /// transition from the active to inactive state (according to the [iOS application lifecycle]).
     ///
-    /// Also see [`Resumed`] notes.
+    /// [`applicationWillResignActive`]: https://developer.apple.com/documentation/uikit/uiapplicationdelegate/1622950-applicationwillresignactive
+    /// [iOS application lifecycle]: https://developer.apple.com/documentation/uikit/app_and_environment/managing_your_app_s_life_cycle
+    ///
+    /// ### Web
     ///
-    /// ## Android
+    /// On Web, the [`suspended()`] method is called in response to a [`pagehide`] event if the
+    /// page is being restored from the [`bfcache`] (back/forward cache) - an in-memory cache that
```

Comment:
I think you meant: `if the page is being moved/stored in(to) the bfcache`?

(EDIT: the original docs said: `the page is being put in the bfcache`)

Reply:
Ah, thanks!

Reply:
Will be fixed in https://github.com/rust-windowing/winit/pull/3786, marking as resolved.